### DISCUSSION TITLE
GitHub Support Wont Delete My Account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#46504] [Other Feature Feedback, Questions, & Ideas] [answered]
So I got caught in this nonsense about having multiple free accounts. I didn't even know it was a crime, but now all my accounts are flagged as spam. I opened a ticket asking them to delete my account (flagged accounts can't be deleted), but instead of doing that they unflagged it and asked me to upgrade by paying $4 a month.

I specifically asked them to delete my account, and they refused. My other account that I want to keep is still flagged. Is there anything I can do to get them to help me?

**Comment by queenofcorgis** [comment, marked as answer]
Hi there 👋🏼 Unfortunately, the community is unable to assist in matters like these, please continue to work with the Support team on the ticket you opened. I'll be locking this post - best of luck!
